modules/_labs/menu/react/package.json

```diff
@@ -35,7 +35,7 @@
     "menu"
   ],
   "devDependencies": {
-    "@material-ui/core": "^4.1.0",
+    "@material-ui/core": "^3.9.2",
```

Comment:
This dependency was never updated in the `yarn.lock` file, so I reverted to a version used by other dependencies. Ideally this is not a dependency at all